Makefile

```diff
@@ -202,6 +202,16 @@ format:
 	@./tools/format_all.sh
 	@make --no-print-directory -C test format
 
+.PHONY: typos
+typos:
+	@cargo install typos-cli
+	@typos
```

Comment:
Will typos find typos under `osdk/` (which is not part of the root Cargo workspace) or `doc/` (which is a Mdbook, not Rust projects) or `test/` (which includes C header and source files)?

Reply:
As I can see in my test, the spelling mistakes found by typos are almost everywhere, including `osdk/`, `doc/` and `test/`.
In my opinion, the typos tool would search for spelling mistakes in each file in current directory, regardless of the structure of this project.

Reply:
Cool.